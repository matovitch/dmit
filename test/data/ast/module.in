module A;

import B;
import C;

export func f() {}
