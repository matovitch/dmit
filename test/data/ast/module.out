{"node":"Module","path":{},"imports":[],"functions":[],"types":[],"modules":[{"node":"Module","path":{"node":"Identifier","lexeme":{"node":"Lexeme","slice":"A"}},"imports":[{"node":"Import","path":{"node":"Identifier","lexeme":{"node":"Lexeme","slice":"C"}}},{"node":"Import","path":{"node":"Identifier","lexeme":{"node":"Lexeme","slice":"B"}}}],"functions":[{"node":"Function","status":"EXPORTED","name":{"node":"Identifier","lexeme":{"node":"Lexeme","slice":"f"}},"arguments":[],"returnType":{},"body":[]}],"types":[],"modules":[]}]}